health/health.go

```diff
@@ -1232,3 +1245,13 @@ func (t *Tracker) checkReceiveFuncsLocked() {
 		f.missing = true
 	}
 }
+
+type MetricHealthMessageLabel struct {
+	Severity string
```

Comment:
Each warnable has a severity as well: https://github.com/tailscale/tailscale/blob/f12aff959aa40b1085dee5def507273538230d65/health/health.go#L217

Even if we are not breaking down warnable counts by severity, we might want to do it in the future. Perhaps call this `Type` instead, i.e. `type="warning"`.